display/startOverview.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+TOMOFILE=/home/bar/Projects/tomoscan
```

Comment:
Whoops user home directory path slipped in here. Instead use for instance the path of this script.

```suggestion
DISPLAY_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
```